res/dpfont_settings.json

```diff
@@ -4,7 +4,7 @@
 	"in-glyphs": [
 		"ABCDEFGHIJKLMNOPQRSTUVWXYZ",
 		"abcdefghijklmnopqrstuvwxyz",
-		"0123456789%./-~,;:!?'&()_"
+		"0123456789%./-~,;:!?'&()_+"
```

Comment:
Didn't you have to modify dpfont_flyphs.png as well?

Reply:
Nope. The + was already there.